Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:14
```

Comment:
Fine for now. But in the future we will have our own base image.

Reply:
yes, this is only for testing purpose for test team.